### PR TITLE
Nodereal

### PR DESCRIPTION
**Pull request by ankushnodereal**
## Description
Added Nodereal as node provider
Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

**Review by ankushnodereal — COMMENTED** [review]
https://eth-mainnet.nodereal.io/v1/f497bff22ce74094857d3778c2953d0a

**Review by ankushnodereal — COMMENTED** [review]
https://eth-mainnet.nodereal.io/v1/f497bff22ce74094857d3778c2953d0a

**Review by ankushnodereal — COMMENTED** [review]
https://eth-mainnet.nodereal.io/v1/f497bff22ce74094857d3778c2953d0a

**Review by ankushnodereal — COMMENTED** [review]
add nodereal as node provider

File: CHANGELOG.md [hunk 1]
```diff
@@ -534,3 +531,13 @@ Released with 1.0.0-beta.37 code base.
 ### Fixed
 -  Fix jsonrpc payload and response types (#4743) (#4761)
 -  Allowed more flexibility in typing the overly constrained `provider.disconnect` function (#4833)
+
+## [1.7.3]
```
Comment (jdevcs) [inline CHANGELOG.md:535]: @ankushnodereal  Why there are version bumps, build files and package lock updates in this PR? We dnt include these changes from external PRs, and we include such changes from internal team members during release only.

**Review by jdevcs — CHANGES_REQUESTED** [review]

**Comment by luu-alex** [comment]
Hi there, @ankushnodereal  theres alot of unnecessary file changes and version updating that is irrelevant. if you would like to discuss about nodereal being used please create a discussion. Thanks ill be closing this

**Pull request closed without merging**